projects/packages/forms/src/blocks/contact-form/child-blocks.js

```diff
@@ -23,234 +19,10 @@ import JetpackInput from '../input';
-			},
-			{
-				type: 'block',
-				blocks: [ 'jetpack/field-telephone' ],
-				transform: attributes => createBlock( 'jetpack/field-telephone', attributes ),
-			},
-			{
-				type: 'block',
-				blocks: [ 'jetpack/field-textarea' ],
-				transform: attributes => createBlock( 'jetpack/field-textarea', attributes ),
-			},
-			{
-				type: 'block',
-				blocks: [ 'jetpack/field-checkbox-multiple' ],
-				transform: ( attributes, innerBlocks ) => {
-					let newInnerBlocks = [];
-
-					if ( ! isEmpty( innerBlocks ) ) {
-						const optionBlocks = filter( innerBlocks, ( { name } ) =>
-							startsWith( name, 'jetpack/field-option' )
-						);
-
-						newInnerBlocks = map( optionBlocks, block =>
-							createBlock( 'jetpack/field-option-checkbox', {
-								label: block.attributes.label,
-								fieldType: 'checkbox',
-							} )
-						);
-					} else if ( attributes.options?.length ) {
-						newInnerBlocks = map( attributes.options, option =>
-							createBlock( 'jetpack/field-option-checkbox', {
-								label: option,
-								fieldType: 'checkbox',
-							} )
-						);
-					}
-
-					return createBlock( 'jetpack/field-checkbox-multiple', attributes, newInnerBlocks );
-				},
-			},
-			{
-				type: 'block',
-				blocks: [ 'jetpack/field-radio' ],
-				transform: ( attributes, innerBlocks ) => {
-					let newInnerBlocks = [];
-
-					if ( ! isEmpty( innerBlocks ) ) {
-						const optionBlocks = filter( innerBlocks, ( { name } ) =>
-							startsWith( name, 'jetpack/field-option' )
-						);
-
-						newInnerBlocks = map( optionBlocks, block =>
-							createBlock( 'jetpack/field-option-radio', {
-								label: block.attributes.label,
-								fieldType: 'radio',
-							} )
-						);
-					} else if ( attributes.options?.length ) {
-						newInnerBlocks = map( attributes.options, option =>
-							createBlock( 'jetpack/field-option-radio', {
-								label: option,
-								fieldType: 'radio',
-							} )
-						);
-					}
-
-					return createBlock( 'jetpack/field-radio', attributes, newInnerBlocks );
-				},
-			},
-			{
-				type: 'block',
-				blocks: [ 'jetpack/field-select' ],
-				transform: ( attributes, innerBlocks ) => {
-					if ( ! isEmpty( innerBlocks ) ) {
-						const optionBlocks = filter( innerBlocks, ( { name } ) =>
-							startsWith( name, 'jetpack/field-option' )
-						);
-						attributes.options = map( optionBlocks, b => b.attributes.label );
-					}
-
-					attributes.options = attributes.options?.length ? attributes.options : [ '' ];
-					return createBlock( 'jetpack/field-select', attributes );
-				},
-			},
-			{
-				type: 'block',
-				blocks: [ 'jetpack/field-consent' ],
-				transform: attributes => createBlock( 'jetpack/field-consent', attributes ),
-			},
-			{
-				type: 'block',
-				blocks: [ 'jetpack/field-checkbox' ],
-				transform: attributes => createBlock( 'jetpack/field-checkbox', attributes ),
-			},
-		],
-	},
-	save: () => null,
-	example: {},
-};
```

Comment:
To confirm, it looks like the big FieldDefaults was in this file just for defining the field-file and does not appear to be used elsewhere in the forms code base at this point.

Reply:
Yes. things are way cleaner now 💅